code/modules/mob/living/silicon/robot/death.dm

```diff
@@ -16,14 +16,15 @@
 /mob/living/silicon/robot/death(gibbed)
 	if(stat == DEAD)
 		return
-
+	if(!gibbed)
+		logevent("FATAL -- SYSTEM HALT")
```

Comment:
Could you explain this change?

Reply:
There's an IC log for borgs in their borgUI app, and this adds an entry for when the borg does. Mostly IC fluff. The third screenshot above shows other entries that can show up in the borg's IC log.

Reply:
Should the computer not shutdown as well if the robot is gibbed?